### DISCUSSION TITLE
Q: How to disable Markdown globally?

### DISCUSSION THREAD

**Discussion by vladsf** [eventum/eventum#956] [Ideas]
**Description**  
Hi, 
It seems Markdown is not always optimal for plain-test emails. Is there way to disable it globally?

**Comment by glensc** [comment]
No.

**Comment by glensc** [comment]
You may create an extension to listen new email event, and modify the body before it's stored to database.

**Comment by vladsf** [comment]
Issue view page has 'plain text' toogle. Is that possible to make it default?

**Comment by glensc** [comment]
or make the toggle remember itself, like hidden notes, emails blocks do
